.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: ci
+on: [push, pull_request]
+env:
+  VROOM_VERSION: 0.14.0
+  MAKTABA_VERSION: 1.14.0
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: "setup linux build environment"
+      run: |
+        sudo apt update
+        sudo apt install -y vim-gtk3 wget xvfb xterm
+        wget https://github.com/google/vroom/releases/download/v$VROOM_VERSION/vroom_$VROOM_VERSION-1_all.deb
+        sudo dpkg -i ./vroom_$VROOM_VERSION-1_all.deb
+        git clone -b ${MAKTABA_VERSION} https://github.com/google/vim-maktaba.git $GITHUB_WORKSPACE/../maktaba/
+        git clone https://github.com/google/vim-glaive.git $GITHUB_WORKSPACE/../glaive/
+
+    - name: "echo vim information"
+      run: vim --version
+
+    - name: "set up log files"
```

Comment:
For all the output redirection, was there a reason `xvfb-run script` didn't work like we used in https://github.com/google/vim-maktaba/blob/master/.github/workflows/run-tests.yml?